.changeset/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.6.0/schema.json",
```

Comment:
I guess this attribute was autogenerated? It does not appear on the docs https://github.com/atlassian/changesets/blob/main/docs/config-file-options.md but has the same structure.

Reply:
Yes